rubocop.yml

```diff
@@ -73,7 +73,7 @@ Layout/LineEndStringConcatenationIndentation:
 
 Layout/LineLength:
   AllowedPatterns:
-  - "\\A\\s*(remote_)?test(_\\w+)?\\s.*(do|->)(\\s|\\Z)"
+  - "\\A\\s*((remote_)?test|context|describe|it|pending|skip|xit)(_\\w+)?\\s.*(do|->)(\\s|\\Z)"
```

Comment:
As a counter-point, maybe we _shouldn't_ do this because the "spec" style of tests would prefer nesting contexts to having a long description... 🤔

```ruby
context "really" do
  context "really" do
    it "long" do
```